backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+
+server.port = 5000
+endpoints.env.enabled=true
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
+spring.datasource.username = ${SECRETS_DB_USERNAME}
+spring.datasource.password = ${SECRETS_DB_PASSWORD}
+spring.jpa.hibernate.ddl-auto = update
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
+spring.datasource.url=jdbc:mysql://${SECRETS_DB_URL}?useLegacyDatetimeCode=false&serverTimezone=UTC
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+jwt.secret="a@a.com"
```

Comment:
Seems like this shouldn't be public either?

Reply:
Added to secrets